doc/usage/build.md

```diff
@@ -18,7 +18,7 @@ cd cozy-desktop
 ## Install NPM dependencies
 
 ```
-yarn install
+yarn install && yarn install:all
```

Comment:
What about having `yarn install:all` include `yarn install`?

Reply:
I will look at it when the PR to split travis build is merged: I'd like to avoid to run `yarn install` several times on travis

Reply:
:ok_hand: Also maybe change the header line 18 (not npm-only anymore?)